packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
My concern is that in a real ES6 environment, this would be a TDZ error because your default value of `= a` is trying to access the value from `...a` before it has been initialized, and instead it's picking up the `const a = 1;` instead, which is incorrect. e.g. ideally we'd get

    var a = 1;
    function rest() {
      var b = arguments.length <= 0 || arguments[0] === undefined ? a : arguments[0];
      var a;

      assert.equal(b, 1);
    }

so at least the `a` binding wouldn't pick up the outer `a`.